.github/workflows/docker-unified.yml

```diff
@@ -663,6 +663,23 @@ jobs:
         run: |
           echo "$DATAHUB_VERSION"
           ./smoke-test/smoke.sh
+      - name: store logs
+        if: always()
+        run: |
+          docker ps -a
+          docker logs datahub-gms >& gms.log
+      - name: Upload logs
+        uses: actions/upload-artifact@v2
```

Comment:
We have previously run out of artifact storage, this will accelerate it?

Reply:
wasn't aware of limit on artifact storage. Changed to be only on failure